clients/fides-js/src/lib/tcf/stub.ts

```diff
@@ -19,7 +19,11 @@ const isMessageData = (data: unknown): data is MessageData =>
 export const makeStub = () => {
   const queue: any[] = [];
   const currentWindow = window;
-  let gdprApplies: boolean;
+  /**
+   * Fides modification (PROD#1433): gdprApplies defaults to true
+   * since we only return TCF when gdpr does apply!
+   */
+  let gdprApplies: boolean = true;
```

Comment:
One ask: for our future benefit, let's do this as an override option `fides_tcf_gdpr_applies` and default it to true there. This gives us an out if we need one 👍